pkgs/tools/text/platinum-searcher/default.nix

```diff
@@ -2,20 +2,25 @@
 
 buildGoPackage rec {
   name = "the_platinum_searcher-${version}";
-  version = "2.1.3";
-  rev = "v2.1.3";
+  version = "2.1.5";
+  rev = "v${version}";
 
   goPackagePath = "github.com/monochromegane/the_platinum_searcher";
 
   src = fetchFromGitHub {
     inherit rev;
     owner = "monochromegane";
     repo = "the_platinum_searcher";
-    sha256 = "09pkdfh7fqn3x4l9zaw5wzk20k7nfdwry7br9vfy3vv3fwv61ynp";
+    sha256 = "1y7kl3954dimx9hp2bf1vjg1h52hj1v6cm4f5nhrqzwrawp0b6q0";
   };
 
   goDeps = ./deps.nix;
 
+  preFixup = stdenv.lib.optionalString stdenv.isDarwin ''
+    # fixes cycle between $out and $bin
+    install_name_tool -delete_rpath $out/lib $bin/bin/pt
+  '';
+
```

Comment:
Alright, I don't understand this fully either. But if it works, I guess it is fine for now.